scripts/postNpmInstall.js

```diff
@@ -0,0 +1,10 @@
+// Remove the webpack-dev-server grunt task since we do not use it
+var fs = require('fs');
+var path = require('path');
+
+var taskFile = path.join(__dirname, '..', 'node_modules', 'grunt-webpack', 'tasks', 'webpack-dev-server.js');
```

Comment:
Just tried, no luck. Because `grunt-webpack` does not specify a `main` inside its package.json, require.resolve fails on it. If there's some way to only get the path to a module directory without looking up its entry point, we could use that instead.